scheduler/setup.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import os
+
+import setuptools
+
+# Package metadata.
+
+name = 'google-cloud-scheduler'
+description = 'Cloud Scheduler API API client library'
+version = '0.1.0'
+# Should be one of:
+# 'Development Status :: 3 - Alpha'
+# 'Development Status :: 4 - Beta'
+# 'Development Status :: 5 - Production/Stable'
+release_status = 'Development Status :: 3 - Alpha'
+dependencies = [
+    'google-api-core[grpc] >= 1.4.1, < 2.0.0dev',
+    'enum34; python_version < "3.4"',
+#    'grpc-google-iam-v1 >= 0.11.4, < 0.12dev',
```

Comment:
Why is this one commented out?  Seems like it should be removed, if there is no IAM in this service.

Reply:
Because I made an oops. It shouldn't be needed, I commented out to run tests and verify there wasn't an issue.